scripts/modelscope/t2v_pipeline.py

```diff
@@ -192,7 +193,26 @@ def compute_latents(self, vd_out, cpu_vae='GPU (half precision)', device=torch.d
         return out
 
     # @torch.compile()
-    def infer(self, prompt, n_prompt, steps, frames, seed, scale, width=256, height=256, eta=0.0, cpu_vae='GPU (half precision)', device=torch.device('cpu'), latents=None, skip_steps=0,strength=0,mask=None):
+    def infer(
+        self, 
+        prompt, 
+        n_prompt, 
+        steps, 
+        frames, 
+        seed, 
+        scale, 
+        width=256, 
+        height=256, 
+        eta=0.0, 
+        cpu_vae='GPU (half precision)', 
+        device=torch.device('cpu'), 
+        latents=None, 
+        skip_steps=0,
+        strength=0,
+        mask=None, 
+        is_vid2vid=False,
+        sampler=available_samplers[0].name
+    ):
```

Comment:
Good idea to expand the definitions, it will be easier to track changes this way